src/global.scss

```diff
@@ -33,3 +33,10 @@
  */
 
 @import "./app/app.scss";
+
+@import '~swiper/scss';
+@import '~@ionic/angular/css/ionic-swiper';
+
+.swiper-slide {
+  display: block;
+}
```

Comment:
For whatever reason, I couldn't get this styling to take when put in the tutorial component CSS. Definitely open to alternatives.

Reply:
The Angular generated styles are scoping `.swiper-slide` to be something like `.swiper-slide[_ngcontent-tpm-c145]`, but the `.swiper-slide` elements are being generated within Swiper which do not receive that `_ngcontent-tpm-c145` scope.

You didn't do anything wrong -- it looks like Swiper 7 has a defect with Angular style encapsulation.